src/Mod/Path/PathScripts/post/fanuc_post.py

```diff
@@ -0,0 +1,512 @@
+# ***************************************************************************
+# *   Copyright (c) 2014 sliptonic <shopinthewoods@gmail.com>               *
```

Comment:
Also update copyright info

Reply:
What should I do with the copyright information?  It's copied from Sliptonic's file..  just just update the date, or replace it with my name, or what?

Reply:
add a new line with current year and your name